lib/env/env.c

```diff
@@ -171,3 +172,56 @@ void spdk_delay_us(unsigned int us)
 {
 	return rte_delay_us(us);
 }
+
+#define BDEV_SYS_CPU_DIR "/sys/devices/system/cpu/cpu%u"
+#define BDEV_CORE_ID_FILE "topology/core_id"
+#define BDEV_CPU_PATH_MAX 256
+
+/* Check if a cpu is present by the presence of the cpu information for it */
+static int blockdev_cpu_detected(unsigned lcore_id)
```

Comment:
Assuming the request below works out (just setting all cpuset bits), this function can be removed.  If it is actually required, it should be renamed, since it is not related to blockdevs.  It is also Linux specific, so it at least needs a stubbed FreeBSD version (or ideally a full implementation).